app/src/androidTest/java/mozilla/lockbox/screenshots/ScreenshotsTest.kt

```diff
@@ -42,120 +42,98 @@ open class ScreenshotsTest {
-        Screengrab.screenshot("allset-screen")
+            openMenu()
+            Screengrab.screenshot("app-menu-screen")
 
-        onView(withId(R.id.finishButton)).perform(click())
-        onView(withId(R.id.sortButton))
-                .check(matches(isDisplayed()))
-        Screengrab.screenshot("all-logins-screen")
+            tapLockNow()
+            Screengrab.screenshot("lock-now-screen")
+            pressBack()
+        }
+    }
 
-        selectItem(1)
-        onView(withId(R.id.inputHostname))
-                .check(matches(isDisplayed()))
-        Screengrab.screenshot("item-detail-screen")
-
-        onView(withId(R.id.kebabMenuButton)).perform(click())
-        Screengrab.screenshot("item-menu")
-
-        onView(withText(R.string.delete)).perform(click())
-        Screengrab.screenshot("item-delete-disclaimer")
-
-        onView(withText(R.string.cancel)).perform(click())
-
-        // Detail credential view
-        onView(withId(R.id.inputUsername)).perform(click())
-        Screengrab.screenshot("username-copied-screen")
-
-        onView(withId(R.id.inputPassword)).perform(click())
-        Screengrab.screenshot("password-copied-screen")
-
-        //  Edit Menu
-        onView(withId(R.id.kebabMenuButton)).perform(click())
-        onView(withText(R.string.edit)).perform(click())
-        Screengrab.screenshot("item-edit-menu")
-
-        // Get error diaglog removing all password
-        onView(withId(R.id.inputPassword)).perform(replaceText(""))
-        // need this or the message will not be caught
-        sleep(2000)
-        Screengrab.screenshot("error-empty-field")
-
-        // Cancel edit menu
-        // Disabled until tapping on 'x' button is available without using text
-        // onView(withId(android.R.id.button1)).perform(click())
-        // Screengrab.screenshot("item-edit-menu-disclaimer")
-        // Temporary solution
-        onView(withId(R.id.saveEntryButton)).perform(click())
-
-        // Sort menu
-        onView(withId(R.id.sortButton)).perform(click())
-        Screengrab.screenshot("sorting-options-screen")
-        pressBack()
-
-        // App Menu
-        onView(withId(R.id.appDrawer)).perform(DrawerActions.open())
-        Screengrab.screenshot("app-menu-screen")
-
-        onView(withId(R.id.lockNow)).perform(click())
-        Screengrab.screenshot("lock-now-screen")
-        pressBack()
-
-        // Settings screens
-        tapSettings()
-        onView(withId(R.id.settingList))
-                .check(matches(isDisplayed()))
+    @Test
+    fun testAppSettingsMenu() {
+        Screengrab.setDefaultScreenshotStrategy(UiAutomatorScreenshotStrategy())
+        navigator.gotoSettings()
         Screengrab.screenshot("settings-menu-screen")
-        pressBack()
 
-        tapAccountSetting()
-        onView(withId(R.id.disconnectButton))
-                .check(matches(isDisplayed()))
+        navigator.gotoAccountSetting()
```

Comment:
This happens after another `navigator` call. Does this work?

Reply:
Yeah, that's working I'm getting the screenshots on both screens successfully but I can change that part if that's bettter so that there are not two calls to navigator one after another